.travis.yml

```diff
@@ -73,6 +79,21 @@ install:
   # make sure the sqlite file exists to avoid race conditions
   - python -c "from bluesky.utils import get_history; get_history()"
 
+before_script:
+  - |
+    if [ "$TRAVIS_OS_NAME" == "osx" ]; then
+        brew cask install homebrew/cask-versions/adoptopenjdk8
+        brew install kafka
+        brew services start zookeeper
+        brew services start kafka
+        sleep 60
+        kafka-topics --create --zookeeper localhost:2181 --replication-factor 1 --partitions 1 --topic macos_test
```

Comment:
Shouldn't we use the same topics as in the https://github.com/bluesky/bluesky/pull/1209/files#diff-99faaf2f932161e4fa2b70f8073c3e42R13 step?

Reply:
there different test topic for macOS is a temporary test that I haven't removed yet